.travis.yml

```diff
@@ -1,8 +1,6 @@
 language: node_js
 
 node_js:
-  - 8
-  - 10
   - 12
```

Comment:
Node 14 just release, maybe we could add it as well

Reply:
```suggestion
  - 12
  - 14
```